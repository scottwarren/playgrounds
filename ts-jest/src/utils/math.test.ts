describe('Math', () => {
  it('add', async () => {
    const result = await import('./math').then(({add}) => add(1, 2, 3));
    expect(result).toEqual(6);
    expect(result).toMatchSnapshot('snap 1');
  });
});
