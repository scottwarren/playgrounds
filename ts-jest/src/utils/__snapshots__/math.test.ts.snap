// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Math add: snap 1 1`] = `6`;
